debian/changelog

```diff
@@ -1,8 +1,116 @@
-glib2.0 (2.46.2-0) master; urgency=medium
+glib2.0 (2.48.0-0) master; urgency=medium
```

Comment:
I think that may be correct when the latest changelog entry has a lower version number than the git tag. However, in this case the git tag is 2.48.0, Jenkins adds -0, and the latest entry is 2.48.0-1, so when I try your suggestion of removing the bogus changelog entry and building with jenkins-obs, I get this error:
```
dch: fatal error at line 1106:
New version specified (2.48.0-0) is less than
the current version number (2.48.0-1)!  Use -b to force.
```
I'd like to go ahead and push the branch including the bogus changelog entry.